### ISSUE TITLE
Incorrect min and max for vectored signals

### ISSUE THREAD

**Issue by bboettcher3** [libmapper/libmapper#89]
There's some ambiguity in supplying min and max values for signals with length > 1 that has propogated some bugs through to webmapper.

The reference for mpr_sig_new() implies a single value for min and a single value for max:
```
 *  \param minimum          Pointer to a minimum value, or 0 for none.
 *  \param maximum          Pointer to a maximum value, or 0 for none.
```

Yet in mpr_sig_init() in signal.c we see the property being set with the same length as the signal:
```
    if (min)
        mpr_tbl_set(tbl, PROP(MIN), NULL, len, type, min, LOCAL_MODIFY);
    if (max)
        mpr_tbl_set(tbl, PROP(MAX), NULL, len, type, max, LOCAL_MODIFY);
```

Finally, this results in incorrect range displays in webmapper due to overlapping table entries when only 1 min/max is supplied:
![image](https://user-images.githubusercontent.com/13266420/204400508-cd1a9c2b-51c3-4252-94d5-f51402570a0b.png)

Should each element in the signal vector have its own range, or should there just be one?

**Comment by malloch** [comment]
The signal constructor is expecting the `min` and `max` arguments (if provided) point to arrays of the same length as the signal. The property setter allows you to set extrema for a vector signal with a scalar or shorter array if desired, e.g.:

`mpr_obj_set_prop((mpr_obj)my_sig, MPR_PROP_MIN, NULL, 1, MPR_FLT, &val, 1));`

I will update the C API documentation to clarify this!

**Comment by bboettcher3** [comment]
Likewise updated for the T-Stick here: https://github.com/IDMIL/T-Stick/pull/25
Will leave this open until documentation is fixed.

**Comment by malloch** [comment]
Fixed in commit 0fe9bce
